.vsts-ci.yml

```diff
@@ -14,7 +14,7 @@ trigger:
 pr: none
 
 queue:
-  name: VSEng-MicroBuildVS2017
+  name: VSEngSS-MicroBuild2019
```

Comment:
The build broke because the host probably doesn't have the right cached docker image, but odd it couldn't pull it? Maybe no connection to the outside world?

```
Step 1/6 : FROM microsoft/windowsservercore@sha256:431315d3208b79a74c7e598aacb469f0e58a18977f5b93885b1d8d7027cae03d
manifest for microsoft/windowsservercore@sha256:431315d3208b79a74c7e598aacb469f0e58a18977f5b93885b1d8d7027cae03d not found: manifest unknown: manifest unknown
```

@lwillia updated these recently and may remember where to look off hand.

Additionally, make sure that the right Windows SDK targets are installed, though I see the `Build` step completed successfully so it would seem it is installed.